psyneulink/core/compositions/composition.py

```diff
@@ -8769,38 +9201,52 @@ def execute(
-                the shape of which must match the node's default variable.
+                the user. For each pair, the key is the Node <Compositon_Nodes>` (A `Mechanism <Mechanism>` or
+                `Composition`) and the value is an input, the shape of which must match the Node's default variable.
+                if **inputs** is not specified, the `default_variable <Component_Variable>` for each `INPUT` Node
+                is used as its input;  see
 
-            scheduler : Scheduler
+            clamp_input : SOFT_CLAMP : default SOFT_CLAMP
+
+            runtime_params : Dict[Node: Dict[Parameter: Tuple(Value, Condition)]] : default None
+                specifies alternate parameter values to be used only during this `EXECUTION` when the specified
+                `Condition` is met. See `Run_Runtime_Parameters` for more details and examples of valid dictionaries.
+
+            skip_initialization :  : default False
+
+            scheduler : Scheduler : default None
                 the scheduler object that owns the conditions that will instruct the execution of the Composition
                 If not specified, the Composition will use its automatically generated scheduler.
 
-            context
-                context will be set to self.default_execution_id if unspecified
+            context : `Context.execution_id>` : default `default_execution_id`
+                context in which the `Composition` will be executed;  set to self.default_execution_id ifunspecified.
 
-            base_context
+            base_context : `Context.execution_id>` : Context(execution_id=None)
                 the context corresponding to the execution context from which this execution will be initialized,
                 if values currently do not exist for **context**
 
-            call_before_time_step : callable
+            call_before_time_step : callable : default None
                 called before each `TIME_STEP` is executed
                 passed the current *context* (but it is not necessary for your callable to take)
 
-            call_after_time_step : callable
+            call_after_time_step : callable : default None
                 called after each `TIME_STEP` is executed
                 passed the current *context* (but it is not necessary for your callable to take)
 
-            call_before_pass : callable
+            call_before_pass : callable : default None
                 called before each `PASS` is executed
                 passed the current *context* (but it is not necessary for your callable to take)
 
-            call_after_pass : callable
+            call_after_pass : callable : default None
                 called after each `PASS` is executed
                 passed the current *context* (but it is not necessary for your callable to take)
 
+            bin_execute : bool or Enum[LLVM|LLVMexec|LLVMRun|Python|PTXExec|PTXRun] : default Python
```

Comment:
I think the `*Run` options should be dropped from here as well. Don't do anything when passed to `execute`.

Reply:
Updated